superset/utils/decorators.py

```diff
@@ -191,3 +191,22 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
 
 def on_security_exception(self: Any, ex: Exception) -> Response:
     return self.response(403, **{"message": utils.error_msg_from_exception(ex)})
+
+
+@contextmanager
+def suppress_logging(
```

Comment:
Not blocking: could we add some tests here? https://github.com/apache/superset/blob/master/tests/unit_tests/utils/test_decorators.py